bin/mink-test-server

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# fail hard
+set -o pipefail
+# fail harder
+set -eu
+
+base_dir=$(dirname "$(dirname "$(readlink -f "$0")")")
+host="localhost:8002"
+php_bin=${MINK_PHP_BIN-php}
```

Comment:
What `${MINK_PHP_BIN-php}` do?

I saw in https://github.com/minkphp/MinkSelenium2Driver/pull/246 the `MINK_PHP_BIN` env variable is defined, but what `-php` part does to it?